Authentication Methods Registration Campaign Graph Docs (#12112)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/authenticationmethodsregistrationcampaign.md

---
title: "authenticationMethodsRegistrationCampaign resource type"

> description: "Settings to run campaigns to push users to setup targeted authentication methods."

Reviewer: ```suggestion
description: "Represents the settings used to run campaigns to push users to set up targeted authentication methods."
```

author: "mjsantani"
localization_priority: Normal

> ms.prod: "microsoft-identity-platform"

Reviewer: ```suggestion
ms.prod: "identity-and-sign-in"
```

doc_type: resourcePageType
---

# authenticationMethodsRegistrationCampaign resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> The Authentication Methods Registration Campaign provides a tool to admins to offer users to setup targeted authentication methods. Users are nudged to setup the authentication method after they successfully complete a MFA challenge. This is currently scoped only to the Microsoft Authenticator app for MFA.

Reviewer: ```suggestion
Represents a tool that administrators can use to remind users to set up targeted authentication methods. Users are reminded to set up the authentication method after they successfully complete a MFA challenge. Only available for the Microsoft Authenticator app for MFA.
```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |excludeTargets|[excludeTarget](../resources/excludetarget.md) collection|Users and groups of users that are excluded from being nudged to setup the authentication method.|

Reviewer: ```suggestion
|excludeTargets|[excludeTarget](../resources/excludetarget.md) collection|Users and groups of users that are excluded from being reminded to set up the authentication method.|
```

|includeTargets|[authenticationMethodsRegistrationCampaignIncludeTarget](../resources/authenticationmethodsregistrationcampaignincludetarget.md) collection|Users and groups of users that will be nudged to setup the authentication method.|

> |snoozeDurationInDays|Int32|Defines after how many days the user will see the nudge again if the user taps on "Not now" and snoozes the prompt. Minimum 0 days. Maximum: 14 days. If value is “0” – User is nudged during every MFA attempt.|

Reviewer: ```suggestion
|snoozeDurationInDays|Int32|Specifies the number of days that the user sees a reminder again if they select "Not now" and snoozes the prompt. Minimum 0 days. Maximum: 14 days. If the value is “0” – The user is reminded during every MFA attempt.|
```

> |state|advancedConfigState|Setting used to enable/disable the feature. Possible values are: `default`, `enabled`, `disabled`, `unknownFutureValue`. The "default" value is used when the cnofiguration hasn't been explicitly set and will use Azure AD's default behavior for the setting. The "default" value currently maps to 'disabled'.|

Reviewer: ```suggestion
|state|advancedConfigState|Represents the setting used to enable or disable the feature. Possible values are: `default`, `enabled`, `disabled`, `unknownFutureValue`. The "default" value is used when the configuration hasn't been explicitly set and uses Azure AD's default behavior for the setting. The default value is 'disabled'.|
```
